acceptance_test.ps1

```diff
@@ -66,34 +62,34 @@ function RunSuite
         foreach ($f in $Framework) {
 
             # Fix 40
-            TestSpec $Configuration $f 5001 "definitions/server/fix40/*.def" "cfg/at_40.cfg" "AcceptanceTests_40_$f.xml" -UseWsl:$($UseWsl -and $UseWsl.IsPresent)
+            TestSpec $Configuration $f "definitions/server/fix40/*.def" "cfg/at_40.cfg" "AcceptanceTests_40_$f.xml" -UseWsl:$($UseWsl -and $UseWsl.IsPresent)
```

Comment:
I was just thinking yesterday how there was no reason the script/app couldn't figure out the port number instead of making the user supply it, so it's nice to see this.